tests/mcp/test_search_cache.py

```
@@ -0,0 +1,262 @@
+        calls["n"] += 1
+        return await orig(g, project)
+
+    monkeypatch.setattr(S, "_build_corpus", counting)
+    return calls
+
+
+async def test_warm_call_reuses_corpus(synth_graph, monkeypatch):
+    import api.mcp.tools.structural as S
+    from api.graph import AsyncGraphQuery
+
+    name, project, branch = synth_graph
```

Ruff (0.15.15)

[warning] Unpacked variable `branch` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)